services/httpd/handler.go

```diff
@@ -913,8 +913,7 @@ func (h *Handler) logging(inner http.Handler, name string) http.Handler {
 		start := time.Now()
 		l := &responseLogger{w: w}
 		inner.ServeHTTP(l, r)
-		logLine := buildLogLine(l, r, start)
-		h.Logger.Println(logLine)
+		fmt.Println("[httpd] " + buildLogLine(l, r, start))
```

Comment:
Ah, I should read the PR first ;)

We should probably just create a logger without the log.Ldate flag if we just need to remove the date from the log. This way it will end up with the rest of the logs and not in STDOUT or something else.